content/deprecated-components/markdown-editor/react/deprecated.mdx

```diff
@@ -0,0 +1,165 @@
+---
+title: Markdown editor
+description: Full-featured Markdown input.
+reactId: markdown_editor
+reactStatus: deprecated
+---
+
+import ReactComponentPage from '~/src/components/react-component-page'
+import { graphql } from "gatsby"
+
+export const query = graphql`
+  query MarkdownEditorDeprecatedQuery($componentId: String! = "markdown_editor", $parentPath: String! = "/deprecated-components/markdown-editor", $status: String! = "deprecated") {
+    ...ReactComponentInfo
+  }
+`
+
+export default function Layout({data, children}) {
+  return <ReactComponentPage data={data} showImport={false} tocItems={[{url: "#import", title: "Import"}, {url: "#examples", title: "Examples"}]}>{children}</ReactComponentPage>
+}
+
+`MarkdownEditor` is a full-featured editor for GitHub Flavored Markdown, with support for:
+
+- Formatting (keyboard shortcuts & toolbar buttons)
+- File uploads (drag & drop, paste, click to upload)
+- Inline suggestions (emojis, `@` mentions, and `#` references)
+- Saved replies
+- Markdown pasting (ie, paste URL onto selected text to create a link)
+- List editing (create a new list item on `Enter`)
+- Indenting selected text
+
+## Import
+
+```js
+import {MarkdownEditor} from '@primer/react/drafts'
```

Comment:
It's in the `drafts` package, not `deprecated`?

Reply:
That's correct 🤦